plugins/module_utils/foreman_helper.py

```diff
@@ -393,12 +393,12 @@ def parse_template_from_file(file_name, module):
 
 
 # Helper for titles
-def split_fqn(title):
+def split_fqn(title, separator='/'):
     """ Split fully qualified name (title) in name and parent title """
-    fqn = title.split('/')
+    fqn = title.split(separator)
```

Comment:
```suggestion
    fqn = title.split(separator, 1)
```

then you don't need to `join` anymore :)

Reply:
Thanks for the pointer :), but I reverted this change as I don't use split_fqn() anymore.